EmbeddedScripts.Shared.Tests/ContainerTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using Xunit;
+
+namespace EmbeddedScripts.Shared.Tests
+{
+    public class ContainerTests
```

Comment:
need at least one more tests for custom type registration (not only primitives like ints and strings)